### DISCUSSION TITLE
Keycloak connection with tls enable database

### DISCUSSION THREAD

**Discussion by ShristyBhati21** [keycloak/keycloak#23304] [Q&A]
Hello there,
I have been trying to connect the keycloak to a custom database that is ssl/tls enabled. The problem I am facing is when I add the custom configuration to add my database which is ssl/tls enabled, the keycloak container is failing and giving me - Path does not chain with any of the trust anchors.
I am new to keycloak so any help or pointers will be appreciated!!
Thanks

**Comment by justin-tay** [comment]
Did you configure the CA cert in the truststore? https://www.keycloak.org/server/keycloak-truststore
